packages/@apphosting/adapter-angular/rollup.config.js

```diff
@@ -0,0 +1,12 @@
+import resolve from "@rollup/plugin-node-resolve";
+import json from "@rollup/plugin-json";
+import commonjs from "rollup-plugin-commonjs";
+
+export default {
+  input: "./dist/simple-server/server.js",
+  output: {
+    file: "./dist/simple-server/bundled_server.mjs",
```

Comment:
Given this output file path has changed, does https://github.com/FirebaseExtended/firebase-framework-tools/blob/main/packages/%40apphosting/adapter-angular/src/utils.ts#L218 still work?

Reply:
It should ultimately end up in the same spot in the dist directory after this change. Rollup was run manually ( before tsc) it was then copied over by tsc (`src/simple-server/*`), now that I've converted simple-server/server to typescript (and changed to `src/simple-server/*.ts`) tsc will produce the .js file in the dist directory and we can do the rollup post-tsc by reaching into that directory.

Reply:
That makes sense to me in theory, but do you mind also giving it a run on an angular-csr app also? (Maybe you've already done this and it works fine) We don't really have unit test coverage around here so those are unlikely to find the issue and I want to be a bit more careful now that we've launched

Reply:
SG. I'll give it a manual verification

Reply:
Added functional test coverage over the adapter